app/sidekiq/form1010cg/submission_job.rb

```diff
@@ -39,7 +43,11 @@ def perform(claim_id)
       log_exception_to_sentry(e)
       StatsD.increment("#{STATSD_KEY_PREFIX}retries")
 
-      increment_applications_retried(claim_id)
+      if Flipper.enabled?(:caregiver1010)
+        StatsD.increment("#{STATSD_KEY_PREFIX}applications_retried") if job_metadata['retry_count'].zero?
```

Comment:
Only increment when the job has raised an error and the `retry_count` is 0. This will replace the redis flag the previous implementation is using.